SparkPost/2web2ui
# FE-899 - Component Wrapper for WindowEvent

[FE-899](https://jira.int.messagesystems.com/browse/FE-899)

### What Changed
- Implements component wrapper for Matchbox `<WindowEvent/>`
- Adds supporting unit tests
- Replaces instances of `WindowEvent`

### How To Test
- Verify `WindowEvent` instances continue to work as expected (for example, the escape key closing the date picker on `/reports/summary`)
- Verify all instances of `WindowEvent` being directly imported were removed (outside of `src/components/matchbox`)

### To Do
